common/src/main/java/com/microsoft/identity/common/internal/ui/webview/certbasedauth/UserChoiceDialog.java

```diff
@@ -94,25 +94,10 @@ public void onCancel(DialogInterface dialog) {
         mDialog = dialog;
     }
 
-    /**
-     * Should dismiss dialog and call the appropriate methods to help cancel the CBA flow.
-     */
-    @Override
-    void onCancelCba() {
```

Comment:
Sorry, definitely something I missed, but how do we call this cancel method to dismiss the dialog, again?

Reply:
Ahh, through SmartcardRemovalPromptDialog

Reply:
onCancelCba() was originally meant for calling some cleaning logic within the challenge handlers when the CBA flow was cancelled unexpectedly (unplugging the USB smartcard early). The name and purpose of this method is pretty confusing... so I instead decided to create the disconnection callback within the smartcard managers, and the challenge handlers could set and clear that callback as needed. Hoping this helps alleviate some of the confusion.

Reply:
I ended up renaming this method to `onUnexpectedUnplug()`. Hopefully this helps some of the confusion.